package.json

```diff
@@ -29,6 +29,7 @@
     "fs-promise": "0.3.1",
     "lodash": "3.10.1",
     "when": "3.7.2",
+    "which": "^1.2.4",
```

Comment:
all other dependencies are pinned to exact versions so this probably should be too